memorials/templates/memorials/memorial.html

```diff
@@ -11,12 +11,12 @@ <h1>
         {{ page.title }}
     </h1>
 
-    <div class="container">
+    <div class="container my-4">
```

Comment:
Should this be `mb-4`?

```suggestion
    <div class="container mb-4">
```